src/cli.ts

```diff
@@ -1,19 +1,21 @@
-#!/usr/bin/env node
```

Comment:
:eyes: This shebang is a strict requirement for this to function as a CLI. Is there any way to preserve this?

Reply:
Hm, it'll still function as a CLI if it's invoked as `node path/to/script`, but obviously not if invoked directly as an executable, i.e. `./path/to/script`.

I could add an override in the ESLint config. That's what I've done in the past. Also, if `cli.js` is intended to be the CLI entry point, should we use the [`bin` field in package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bin) for that?

Reply:
We are already using the `bin` field.

Having to invoke this using `node` would be very inconvenient, and would break CI in the extension for example.

Which rule is preventing this from being used? :thinking:

Reply:
I just noticed we were already using the `bin` field as you replied 😓 

It's `node/shebang` that prevents it, because `cli.ts` is not the file that the package.json `bin` field is pointing to, I'm guessing. Either way, it's restored in 4a08eed, and I confirmed the shebang is there in `dist/cli.js` as well.

Reply:
Ahhh I see, makes sense :+1: